promscale/send-data/jaeger.md

```diff
@@ -0,0 +1,51 @@
+# Send Jaeger traces to Promscale
+Promscale natively supports the OpenTelemetry Line Protocol (OTLP) for traces.
```

Comment:
```suggestion
Promscale natively supports the OpenTelemetry protocol (OTLP) for traces.
```

Reply:
From the OTLP docs: 
![image](https://user-images.githubusercontent.com/3914967/165223535-840e1577-c7fe-450a-994e-abde12af9d4d.png)